hazelcast/src/main/java/com/hazelcast/core/server/StartServer.java

```diff
@@ -33,6 +37,16 @@ private StartServer() {
      * @param args none
      */
     public static void main(String[] args) {
-        Hazelcast.newHazelcastInstance(null);
+        HazelcastInstance hz = Hazelcast.newHazelcastInstance(null);
+        if (System.getProperty("member.tag") != null) {
```

Comment:
This is something that should be pulled into its own method. E.g printMembersTag or something.